src/test/resources/testResult/collections.json

```diff
@@ -0,0 +1,75 @@
+{ "collections" : [
+  {
+    "collectionId":"AHS",
+     "itemType":"feature",
+      "title":"Arkansas Hot Springs National Park Network",
+       "description":null,
+       "keywords":[""],
+       "extent":{
+         "spatial" :
+           {"bbox" : [
+             [
```

Comment:
It is interesting that the collections output has different bbox data than the individual collection output. This will raise questions.